ReleaseConfig

```diff
@@ -37,7 +37,7 @@ version_tuple = (4, 8, 2, 'a', 0)
 # when that version is used.  Python versions prior to deprecate_python_version
 # cause a warning to be issued (assuming it's not disabled).  These values are
 # mandatory and must be present in the configuration file.
-unsupported_python_version = (3, 6, 0)
+unsupported_python_version = (3, 7, 0)
 deprecated_python_version = (3, 7, 0)
```

Comment:
This one has to bump too. The logic is simple but counter-intuitive - it actually represents the lowest version that isn't deprecated. 
```py
def python_version_deprecated(version=sys.version_info):
    return version < deprecated_python_version
```